### PR TITLE
[4929] - double loader in checkout summary block on mobile - fixed

### PR DESCRIPTION
**Pull request by boltzmannsBane**
**Related issue(s):**
* Fixes scandipwa/scandipwa/issues/4929
* Fixes scandipwa/scandipwa/issues/5013
 
**Problem:**
* Since on mobile the order summary and product quantity are the same element, their loaders would overlap.

**In this PR:**
* Hid one of the loaders.

### CODE REVIEW

File: packages/scandipwa/src/component/CheckoutOrderSummary/CheckoutOrderSummary.style.scss [hunk 1]
```diff
@@ -17,6 +17,12 @@
 .CheckoutOrderSummary {
     $divider-border: 1px solid var(--input-border-color);
 
+    @include mobile {
+        > .Loader {
+            display: none;
+        }
+    }
```
Comment (zans-laksa) [inline packages/scandipwa/src/component/CheckoutOrderSummary/CheckoutOrderSummary.style.scss:24]: Or maybe we shouldn't render this loaded at all? :)

**Review by zans-laksa — CHANGES_REQUESTED** [review]

File: packages/scandipwa/src/component/CheckoutOrderSummary/CheckoutOrderSummary.style.scss (hunk 1 above)
Comment (leonazava) [reply packages/scandipwa/src/component/CheckoutOrderSummary/CheckoutOrderSummary.style.scss:24]: it's not rendered with display: none. i think there's smth like "isMobile" prop in javascript but using it yields the same result as these CSS rules. i agree that i don't see the utility in this loader, but it was requested by QA [here](https://github.com/scandipwa/scandipwa/issues/4078). @carinadues what do you think, should we maybe just remove it altogether for all devices?

**Review by zans-laksa — APPROVED** [review]

**Comment by leonazava** [comment]
should be fixed in [4984](https://github.com/scandipwa/scandipwa/pull/4984)

**Comment by Elena-Evsa** [comment]
1. Plus/minus absent in cart
image
2. Cross / delete functionality present in Summary on Checkout
image
3. 4929 - shown 1 loader in case if user login on shipping step and Qty of products changed

**Review by zans-laksa — APPROVED** [review]

**Comment by Elena-Evsa** [comment]
3. Endless loading after user added product to cart (https://pr-4978.d25kd40decjpue.amplifyapp.com/default/monitor-grouped.html)


Also please merge master to pr

**Comment by Elena-Evsa** [comment]
Issue 1:  passed
Issue 2: Cross still present in Summary on checkout 
Issue 3: Passed

**Pull request closed without merging**